### ISSUE TITLE
Sea Harrier FA.2 ALARM anti-radiation missile loadout

### ISSUE THREAD

**Issue by Anon9944** [PygmalionOfCyprus/cmo-db-requests#2065]
### DB Selector

DB3K

### User(s)

Aircraft 257, Aircraft 1148

### Weapons Carried

- 2x AIM-120B AMRAAM
- 2x190 ImpG Drop Tank
- 2x ALARM

### Requires Buddy Illumination

No

### Default Combat Radius (nm)

385nm

### Weather Restrictions

All-Weather

### Time-of-Day Restrictions

Day & Night

### Capable of Quick Turnaround

Yes

### Comments

SEAD loadout

### Sources

https://navalairhistory.com/2012/10/15/sea-harriers-in-the-med-1995/

"The FA.2 could also carry the AIM-9 Sidewinder short-range air-to-air missile, the Sea Eagle anti-ship missile and the ALARM anti-radiation missile."

https://www.naval-technology.com/projects/fa2-sea-harrier/

"The Sea Harrier can carry the MBDA ALARM anti-radiation missile,"

**Comment by FrangibleCover** [comment]
https://www.armadainternational.com/2021/03/when-they-sounded-the-alarm/

"MBDA’s statement put the lack of export success down to the fact that ALARM had only been integrated on one airframe, the Panavia Tornado, compared to competing weapons like the AGM-88 which could outfit a plethora: “This made it a challenge to export, despite its advantages and proven operational capability.”"

Unfortunately, ALARM was never used operationally on Sea Harrier. It was trialled, but never funded for proper integration due to the budget cuts of the 90s. This loadout is what would be used if the ALARM was carried, and I think it would be cleared for service in the event that it was operationally required, but it was never actually flown.

**Comment by bostonmyk** [comment]
Yeah Alarm was never integrated operationally. Looked into this a couple times.
